src/Compilers/CSharp/Test/Semantic/Semantics/ReadOnlyStructsTests.cs

```diff
@@ -1203,14 +1245,14 @@ public void ReadOnlyProperty_RedundantReadOnlyAccessor()
             var csharp = @"
 public struct S
 {
-    public readonly int P { readonly get; }
+    public readonly int P { readonly get => 42; }
```

Comment:
> => 42 [](start = 40, length = 6)

Why did you need to add this?

Reply:
The prototype used to allow putting readonly on an auto property but the spec changed to disallow it.

Reply:
We're now going to allow `readonly` getter-only properties and `readonly` get accessors but there's no need to change this test back.